File: C#-CarRacingProject/Domain/Staff.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.CompilerServices;
using System.Text;
using System.Threading.Tasks;

namespace ProiectCSharpVersion.Domain
{
    class Staff:Entity<Tuple<string,string>>
    {
        public string Username { get; set; }

        public string Password { get; set; }
        public string Nume { get; set; }

        public Staff(string username, string password, string nume)
        {
            this.Nume = nume;
            this.Username = username;
            this.Password = password;
            this.Id = new Tuple<string,string>(this.Username,this.Password);

        }

        public override string ToString()
        {
            return "Staff: nume=" + Nume;
        }
    }
}

File: CarRacing_Client_Server_Java/CurseModel/src/main/java/curse/model/Staff.java
package curse.model;

public class Staff extends Entity<Tuple<String,String>>{
    private final String username;
    private final String password;
    private String nume;

    public Staff(String nume,String username,String password) {
        this.nume=nume;
        this.username=username;
        this.password=password;
        this.setId(new Tuple<>(this.username,this.password));
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    @Override
    public String toString() {
        return "Staff{"+
                "username="+username+
                " nume="+this.nume+"}";
    }
}
